data/json/monsters/slimes.json

```
@@ -173,6 +173,6 @@
     "harvest": "exempt",
     "special_attacks": [ [ "SLIMESPRING", 15 ] ],
     "death_function": [ "ACID" ],
-    "flags": [ "SEES", "HEARS", "SMELLS", "GOODHEARING", "NOHEAD", "POISON", "VENOM", "WARM", "GUILT" ]
+    "flags": [ "SEES", "HEARS", "SMELLS", "GOODHEARING", "NOHEAD", "POISON", "VENOM", "WARM", "GUILT", "PATH_AVOID_DANGER_1" ]
```

```suggestion
    "flags": [ "SEES", "HEARS", "SMELLS", "GOODHEARING", "NOHEAD", "POISON", "VENOM", "WARM", "GUILT", "PATH_AVOID_DANGER_2" ]
```
Turns outt the level-1 version only avoids fire/lava, ledges, and pits, and also some code for only avoiding sharp terrain if not attacking something which I don't know will apply as expected to a friendly monster. If you want it to apply to all dangerous traps the monster can see then you need the level-2 version, but if just avoiding those hazards is good enough to achieve desired effect lemme know.

---

Ah, thanks for catching that! I see no reason not to give it the level-2 version